Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverPool.hs

```diff
@@ -982,9 +989,12 @@ computeActualDistance distanceUnit orgId merchantOpCityId pickup driverPoolResul
           distanceUnit
         }
   logDebug $ "get distance results" <> show getDistanceResults
-  return $ mkDriverPoolWithActualDistResult transporter.defaultPopupDelay <$> getDistanceResults
+  prevRideDropGeoHash <- case prevRideDropLatLn of
+    Just (LatLong lat lon) -> pure $ T.pack <$> DG.encode 10 (lat, lon)
```

Comment:
here encoding with `10 precision`, and while checking encoding with `9 precision` and matching.... 🙃